driver/src/main/scala/api/MongoDriver.scala

```diff
@@ -98,17 +100,18 @@ class MongoDriver(
    * Awaits the termination until the timeout is expired.
    */
```

Comment:
For this one adding the `@deprecated` already seems right

Reply:
done!